app/javascript/submission_handling/polling.js

```diff
@@ -59,11 +66,22 @@ const deleteData = function (jobId) {
 
 export const renderData = (target, resultCallback, otherRule = null) => {
     pollFetch(target, otherRule).then((result) => {
+        if (checkError(result)) {
+            reportError(result.display_error, result.id);
```

Comment:
```suggestion
            reportError(result.display_error);
```
is `result.id` needed?